ci/authn-gcp/get_func_tokens_to_files.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash -ex
+# Script that executes several curl requests to a Google cloud function
+# to get identity tokens with different audience values and writes them to files
+
+GCP_FUNC_URL="https://${GCP_ZONE}-${GCP_PROJECT}.cloudfunctions.net/${GCP_FETCH_TOKEN_FUNCTION}"
+
+main() {
+  echo 'get_func_tokens_to_file.sh'
+  validate_pre_requisites || exit 1
+
+  # Read the identity token file
+  IDENTITY_TOKEN=$(cat $IDENTITY_TOKEN_FILE)
+  export "$IDENTITY_TOKEN"
+
+  test_token_function "conjur/cucumber/host/test-app" || exit 1
+  get_tokens_to_file || exit 1
+  echo '-> get_func_tokens_to_file.sh done'
+}
+
+validate_pre_requisites() {
+  echo 'validate_pre_requisites'
+  if [ ! -f "$IDENTITY_TOKEN_FILE" ]; then
+    echo "Error: identity token file: '$IDENTITY_TOKEN_FILE' file not found."
+    pwd
+    ls -l
+    exit 1
+  fi
+
+  if [ -z "$GCP_PROJECT" ]; then
```

Comment:
weren't these checked already? It might be redundant unless we do an unsetting of env vars

Reply:
Not in this script.